packages/grid_client/tests/modules/compute_capacity.test.ts

```diff
@@ -10,57 +10,57 @@ describe("Compute Capacity module", () => {
     expect(computeCapacity).toBeInstanceOf(ComputeCapacity);
   });
 
-  test("Min values for cpu & memory.", () => {
-    const cpu = 0;
-    const mem = 255 * 1024 ** 2;
+  // test("Min values for cpu & memory.", () => {
```

Comment:
we can skip the test like that
```suggestion
  test.skip("Min values for cpu & memory.", () => {
```